system/system_monitor/include/system_monitor/traffic_reader/traffic_reader_service.hpp

```diff
@@ -69,6 +69,11 @@ class TrafficReaderService
    */
   void start_nethogs(boost::archive::text_iarchive & archive);
 
+  /**
+   * @brief Stop nethogs
+   */
+  void stop_nethogs();
```

Comment:
A Same point.`skip_nethogs`

Reply:
Fixed in fad266d995c46e12f8ff763e4db67cb19a64298d